### ISSUE TITLE
Removed columns: Predicted_fused_transcripts and Predicted_fused_proteins

### ISSUE THREAD

**Issue by pmrotem** [ndaniel/fusioncatcher#79]
Hi,

I noticed that in the current version of FusionCatcher (1.0) the columns Predicted_fused_transcripts and Predicted_fused_proteins were removed from the final table (they existed in version 0.99.6a beta)

I actually used these columns in downstream analysis, and I find them extremely useful and relevant. Why were they removed? Can there be an option (flag) to put these columns in?

Thank you very much,
Rotem

**Comment by ndaniel** [comment]
Hi @pmrotem 

now those can be found in `final-list_candidate-fusion-genes_sequences.txt.zip`.

Cheers,
Daniel

**Comment by pmrotem** [comment]
Hi Daniel,

Thank you for the light-speed response. The columns in the zip file contain the entire transcript sequence. How can I view just the names of the transcripts? In the `fusioncatcher.py` file I noticed there is some flag for `compress-transcripts`, which may be relevant but I don't know how to turn it on.

Furthermore, I also noticed that in the `fusioncatcher.py` file there is another call to `filter-wiggle.py`, which leads to difference between the `final-list_candidate-fusion-genes.txt` and the zip file. Can you explain why?

Thank you,

Rotem

**Comment by ndaniel** [comment]
> The columns in the zip file contain the entire transcript sequence. How can I view just the names of the transcripts? In the fusioncatcher.py file I noticed there is some flag for compress-transcripts, which may be relevant but I don't know how to turn it on.

```
    # predict effect of fusion
    job.add(_FC_+'predict_frame.py',kind='program')
    job.add('--gtf',datadir('organism.gtf'),kind='input')
    job.add('--transcripts',datadir('transcripts.fa'),kind='input')
    job.add('--input',outdir('final-list_candidate-fusion-genes-t3.txt'),kind='input',temp_path=temp_flag)
    job.add('--output',outdir('final-list_candidate-fusion-genes_sequences.txt'),kind='output')
    #fusion_transcripts_sequences:
    #job.add('--compress-transcripts',kind='parameter')
    job.run()
```

should be
```
    # predict effect of fusion
    job.add(_FC_+'predict_frame.py',kind='program')
    job.add('--gtf',datadir('organism.gtf'),kind='input')
    job.add('--transcripts',datadir('transcripts.fa'),kind='input')
    job.add('--input',outdir('final-list_candidate-fusion-genes-t3.txt'),kind='input',temp_path=temp_flag)
    job.add('--output',outdir('final-list_candidate-fusion-genes_sequences.txt'),kind='output')
    #fusion_transcripts_sequences:
    #job.add('--compress-transcripts',kind='parameter')
    job.run()
```

> Furthermore, I also noticed that in the fusioncatcher.py file there is another call to filter-wiggle.py, which leads to difference between the final-list_candidate-fusion-genes.txt and the zip file. Can you explain why?

Indeed that is the case. `filter-wiggle.py` tries to collapse the candidate fusion transcripts which are the same but listed several times (i.e. removing duplicates). Therefore `final-list_candidate-fusion-genes_sequences.txt` has the same number  or more of fusion transcripts as `final-list_candidate-fusion-genes.txt`.

**Comment by pmrotem** [comment]
Hi Daniel,

The two code blocks seem the same to me....

Is there a flag in the command line which can be used? I prefer not to change the code itself...

If there is not flag in the command line, can I request it as a feature?

Thank you!

Rotem

**Comment by ndaniel** [comment]
> The difference between blocks is that this
```
#job.add('--compress-transcripts',kind='parameter')
```
is changed into this:
```
job.add('--compress-transcripts',kind='parameter')
```
by having the # removed.

Yes, it can be added as a feature in the next release.

**Comment by pjongeneel** [comment]
Saw this feature was included in 1.1.0 under the option --compress-transcripts
Just FYI to anyone reading.
